common/beerocks/bcl/source/network/network_utils.cpp

```diff
@@ -419,23 +419,24 @@ static int parseRoutes(struct nlmsghdr *nlHdr, std::shared_ptr<route_info> rtInf
     return (0);
 }
 
-std::string network_utils::get_mac_from_arp_table(const std::string &ipv4)
+static void get_arp_table(std::unordered_map<std::string, std::string> &ip_mac_list, bool ip_key)
 {
-    /**
- * /proc/net/arp looks like this:
- *
- * IP address       HW type     Flags       HW address            Mask     Device
- * 192.168.12.31    0x1         0x2         00:09:6b:00:02:03     *        eth0
- * 192.168.12.70    0x1         0x2         00:01:02:38:4c:85     *        eth0
- */
+    // /proc/net/arp looks like this:
+    // IP address       HW type     Flags       HW address            Mask     Device
+    // 192.168.12.31    0x1         0x2         00:09:6b:00:02:03     *        eth0
+    // 192.168.12.70    0x1         0x2         00:01:02:38:4c:85     *        eth0
     std::ifstream arp_table("/proc/net/arp");
```

Comment:
This is definitely not the way to do this.
If we decide to go this way, should use netlink sockets instead.